src/main/java/com/alkemy/ong/controllers/AuthorizationController.java

```diff
@@ -56,17 +56,15 @@ public ResponseEntity<String> login(@Valid @RequestBody UserLoginDto userLoginDt
      * @throws EmailNotSendException
      */
     @PostMapping("/register")
-    public ResponseEntity<UserEntity> signup(@RequestBody @Valid UserRegisterDto userDTO) throws EmailNotSendException, IOException {
-//        userService.saveUser(userDTO);
-//        Authentication auth;
-//        auth = authenticationManager.authenticate(
-//                new UsernamePasswordAuthenticationToken(userDTO.getEmail(), userDTO.getPassword()));
-//        final String jwt = jwtTokenUtil.generateToken(auth);
-//        return ResponseEntity.ok(new AuthResponseDTO(jwt));
-        UserEntity user = userService.saveUser(userDTO);
-        emailService.sendEmailRegister(user.getEmail());
+    public ResponseEntity<String> signup(@RequestBody @Valid UserRegisterDto userRegisterDto) throws EmailNotSendException, IOException {
+
+        try {
```

Comment:
The exception was already being thrown in the handler, the try cath is not necessary... Then it could be caught with the error handler in the future.

Reply:
Right, it's fixed now!